src/index.js

```diff
@@ -255,6 +255,8 @@ class ChartComponent extends React.Component {
   }
 
   destroyChart() {
+    if (!this.chartInstance) return;
```

Comment:
Can we switch this to this just to match code style?

```suggestion
    if (!this.chartInstance) {
          return;
     }
```

Reply:
@jerairrest done